python/pyspark/sql/functions/builtin.py

```diff
@@ -5494,7 +5494,7 @@ def first(col: "ColumnOrName", ignorenulls: bool = False) -> Column:
     |  Bob|         5|
     +-----+----------+
 
-    Now, to ignore any nulls we needs to set ``ignorenulls`` to `True`
+    Now, to ignore any nulls we need to set ``ignorenulls`` to `True`
```

Comment:
nit: 
```suggestion
    To ignore any null values, set ``ignorenulls`` to `True`
```

Reply:
Also updated for line 5770. Thank you for reviewing. @allisonwang-db